src/com/spikes2212/genericsubsystems/drivetrains/HolonomicDrivetrain.java

```diff
@@ -1,9 +1,12 @@
 package com.spikes2212.genericsubsystems.drivetrains;
 
-import edu.wpi.first.wpilibj.command.Subsystem;
-
 public abstract class HolonomicDrivetrain extends TankDrivetrain {
-	public abstract void holonomicDrive(double speedY, double speedX); // y-forward/backward
-																// x-left/right
 	
+	/**
+	 * @param speedY the speed meant for the backward and forward movement of the drive system
+	 * @param speedX the speed meant for the right and left movement of the drive system

```

Comment:
Should say something here about where positive X should go and such.
Also explain what the method does, not only what the parameters mean